jobs/logs/local/local.go

```diff
@@ -15,20 +15,20 @@ var errorKey = []byte("error")
 // ErrNoLogsForJob is returned when we try to extract the logs of a non existing job
 var ErrNoLogsForJob = errors.New("No logs for job")
 
-// LocalLogWriter contains all that's needed to create a local log writer
-type LocalLogWriter struct {
+// LogWriter contains all that's needed to create a local log writer
+type LogWriter struct {
 	jobID uint64
 }
```

Comment:
Yeah, now that we configure it in a different way it doesn't make much sense anymore.
I'll fix it.

Reply:
Ok, here's the deal, you should build an interface for the logWriter, something like this:

```go
type LogWriter interface {
  Append(string)
}
```

Then you return this instead of the specific local logger:

```go
func NewJobLogWriter(jobID uint64) logWriter {
...
}
```

And then you change the name of your type, like this:

```go
type localLogWriter struct {}
```

Which opens the door for 

```go
type remoteLogWriter struct {}
```

Both implementing the `Append(string)` method.

Then you are not returning something hidden, but the interface instead, and the specific implementation is hidden behind the whatever we are using.